packages/fiori/src/themes/Timeline.css

```diff
@@ -21,3 +21,20 @@
 .ui5-timeline-list-item:last-child {
 	margin-bottom: 0;
 }
+
+:host([axis-orientation="horizontal"]) .ui5-timeline-list {
```

Comment:
After introducing the enum, all CSS selectors should start with capital letter as well

Reply:
Done